docs/_data/portland-2022-schedule.yaml

```diff
@@ -0,0 +1,98 @@
+writing_day:
+  - time: '8:00 <a href="https://time.is/PT">Pacific</a>'
+    title: 'Platform opens'
+  - time: '9:00'
+    title: Morning introduction - Introduction to Sprints & Project Introductions
+  - time: '9:30'
+    title: Team up and start working
+  - time: '12:00'
+    title: Lunch Break
+  - time: '12:30'
+    title: Conference Intro and Tour with the Welcome Wagon
+  - time: '5:00'
+    title: End of Writing Day
+
+talks_day1:
+  - time: '8:00 <a href="https://time.is/PT">Pacific</a>'
+    title: Platform opens
+  - time: '8:30'
+    title: Welcome Wagon conference introduction and platform tour
+  - time: '9:00'
+    title: Write the Docs Team - Community update
+  - time: '9:30'
+    title: "<b>Unconference Starts</b>"
+  - time: '9:30'
+    slug: TBD 1/2
+  - time: '10:00'
+    title: '5 minute break'
+  - time: '10:05'
+    slug: TBD 2/2
+  - time: '10:35'
+    title: '5 minute break'
+  - time: '10:40'
+    title: 'Two talk Q&A'
+  - time: '10:55'
+    title: 'Coffee break'
+  - time: '11:10'
+    slug: TBD 1/2
+  - time: '11:40'
+    title: '5 minute break'
+  - time: '11:45'
+    slug: TBD 2/2
+  - time: '12:15'
+    title: 'Two talk Q&A'
+  - time: '12:30'
+    title: 'Lunch Break'
+  - time: '1:30'
+    title: Lightning Talks (30 mins)
```

Comment:
Rose asked if we could have a break after lightning talks, 5 min should probably be enough

Reply:
That doesn't feel like a problem to me, lightning talks are talks innit ;)

Reply:
I don't think we should cut more of the lightning talks. We already made the conference shorter. We're already using lots of weird start times, I think lunch is the main one to keep on a round number if possible.